src/native/fs/fs_napi.cpp

```diff
@@ -1385,6 +1389,17 @@ fs_napi(Napi::Env env, Napi::Object exports)
 {
     auto exports_fs = Napi::Object::New(env);
 
+    uv_lib_t *lib = (uv_lib_t*) malloc(sizeof(uv_lib_t));
```

Comment:
Define as null first and only malloc inside the condition?

Reply:
Will move both inside the condition